docs/intro/tutorial.rst

```diff
@@ -1,141 +1,446 @@
 .. _intro-tutorial:
 .. _page-objects:
 
-=====================
-web-poet on a surface
-=====================
+========
+Tutorial
+========
 
-In this tutorial, we’ll assume that web-poet is already installed on your
-system.
+In this tutorial you will learn to use web-poet as you write web data
+extraction code for book detail pages from `books.toscrape.com`_.
 
-We are going to scrape `books.toscrape.com <http://books.toscrape.com/>`_,
-a website that lists books from famous authors.
+.. _books.toscrape.com: http://books.toscrape.com/
 
-Creating a Page Object
-======================
+To follow this tutorial you must first be familiar with Python_ and
+:ref:`install web-poet <install>`.
 
-Let's create a Python file where we'll put our first Page Object implementation.
-This Page Object will be responsible for extracting book links from the book
-list page on `books.toscrape.com <http://books.toscrape.com/>`_.
+.. _Python: https://docs.python.org/
 
-.. code-block:: python
 
-    from web_poet.pages import WebPage
+Create a project folder
```

Comment:
I’m applying the change, but for the record I think it is OK to use the word folder even when there is no GUI involved, and I think it can be less confusing to some users.